WooCommerce/Classes/POS/Utils/POSFontStyle.swift

```diff
@@ -5,35 +5,32 @@ import SwiftUI
 enum POSFontStyle {
     case posTitleRegular
     case posTitleEmphasized
-    case posBodyRegular
-    case posBodyEmphasized
-    case posLargeDetailEmphasized
-    case posLargeDetailRegular
-    case posDetailLight
-    case posDetailRegular
-    case posDetailEmphasized
+    case posBodyLargeRegular
+    case posBodyLargeEmphasized
+    case posBodyMediumEmphasized
+    case posBodyMediumRegular
+    case posBodySmallRegular
+    case posBodySmallEmphasized
     case posButtonSymbol
 
     func font(maximumContentSizeCategory: UIContentSizeCategory? = nil) -> Font {
         switch self {
         case .posTitleRegular:
-            Font.system(size: scaledValue(36, maximumContentSizeCategory: maximumContentSizeCategory ?? .accessibilityLarge), weight: .medium)
+            Font.system(size: scaledValue(FontSize.heading, maximumContentSizeCategory: maximumContentSizeCategory ?? .accessibilityLarge), weight: .medium)
         case .posTitleEmphasized:
-            Font.system(size: scaledValue(36, maximumContentSizeCategory: maximumContentSizeCategory ?? .accessibilityLarge), weight: .bold)
-        case .posBodyRegular:
-            Font.system(size: scaledValue(24, maximumContentSizeCategory: maximumContentSizeCategory), weight: .regular)
-        case .posBodyEmphasized:
-            Font.system(size: scaledValue(24, maximumContentSizeCategory: maximumContentSizeCategory), weight: .bold)
-        case .posLargeDetailEmphasized:
-            Font.system(size: scaledValue(20, maximumContentSizeCategory: maximumContentSizeCategory), weight: .semibold)
-        case .posLargeDetailRegular:
-            Font.system(size: scaledValue(20, maximumContentSizeCategory: maximumContentSizeCategory), weight: .regular)
-        case .posDetailLight:
-            Font.system(size: scaledValue(16, maximumContentSizeCategory: maximumContentSizeCategory), weight: .regular)
-        case .posDetailRegular:
-            Font.system(size: scaledValue(16, maximumContentSizeCategory: maximumContentSizeCategory), weight: .medium)
-        case .posDetailEmphasized:
-            Font.system(size: scaledValue(16, maximumContentSizeCategory: maximumContentSizeCategory), weight: .semibold)
+            Font.system(size: scaledValue(FontSize.heading, maximumContentSizeCategory: maximumContentSizeCategory ?? .accessibilityLarge), weight: .bold)
+        case .posBodyLargeRegular:
+            Font.system(size: scaledValue(FontSize.bodyLarge, maximumContentSizeCategory: maximumContentSizeCategory), weight: .regular)
+        case .posBodyLargeEmphasized:
+            Font.system(size: scaledValue(FontSize.bodyLarge, maximumContentSizeCategory: maximumContentSizeCategory), weight: .bold)
+        case .posBodyMediumEmphasized:
+            Font.system(size: scaledValue(FontSize.bodyMedium, maximumContentSizeCategory: maximumContentSizeCategory), weight: .semibold)
```

Comment:
As I understand, all bold/emphasized styles should be `.bold` (weight 700)

```suggestion
            Font.system(size: scaledValue(FontSize.bodyMedium, maximumContentSizeCategory: maximumContentSizeCategory), weight: .bold)
```